ddl/db_test.go

```diff
@@ -2071,6 +2066,10 @@ func (s *testDBSuite3) TestGeneratedColumnDDL(c *C) {
 		{`create table test_gv_ddl_bad (a int, b int, c int as (a+b) primary key)`, mysql.ErrUnsupportedOnGeneratedColumn},
 		{`create table test_gv_ddl_bad (a int, b int, c int as (a+b), primary key(c))`, mysql.ErrUnsupportedOnGeneratedColumn},
 		{`create table test_gv_ddl_bad (a int, b int, c int as (a+b), primary key(a, c))`, mysql.ErrUnsupportedOnGeneratedColumn},
+
+		// add stored generated column through alter table
```

Comment:
```suggestion
		// Add stored generated column through alter table.

Reply:
All of the similar comments in this function are uncapitalized. Do I need to change them together?

Reply:
You can only change the comment in this PR first.

Reply:
Well, I change all of them for consistency. PTAL